src/backends/torch/torchmodule.h

```diff
@@ -52,7 +52,8 @@ namespace dd
     /**
      * \brief forward (inference) pass over the network
      */
-    c10::IValue forward(std::vector<c10::IValue> source);
+    c10::IValue forward(std::vector<c10::IValue> source,
```

Comment:
I'd prefer forward_method as a class parameter, to keep forward signature simple

Reply:
Not really because the use-case is to be able to change the forward method per call. E.g. the CLIP model has `encode_text` and  `encode_image` within the same JIT model.

Reply:
ok